screens/HomeScreen.js

```diff
@@ -189,7 +189,10 @@ export default class HomeScreen extends React.Component {
                 return;
               }
 
-              if ((this.state.serverUrl && !url.startsWith(this.state.serverUrl)) || url.includes('/System/Logs/Log')) {
+                if ((this.state.serverUrl && 
+                        !url.startsWith(this.state.serverUrl) && 
+                        !url.startsWith(this.state.serverUrl.replace(new RegExp("^http://"),"https://"))) 
```

Comment:
A regex literal should be more performant:
```suggestion
                        !url.startsWith(this.state.serverUrl.replace(/^http:\/\//i, "https://"))) 
```

Reply:
Thanks, my syntax highlighting was leading me astray when I was attempting to escape the slashes in a literal like that; I should have trusted myself more than my editor there. Updated!